File: resources/export/java/PIDCommand-source.java
#set($command = $helper.getByName($command-name, $robot))
#set($component = $command.getProperty("Input").getValue())
#set($name = ${helper.getByName($component, $robot).fullName})
#if($name)        return RobotMap.#variable($name).pidGet();
#end

File: src/main/resources/export/cpp/PIDCommand-source.cpp
#set($command = $helper.getByName($command_name, $robot))
#set($sensor = $command.getProperty("Input").getValue())
#set($subsystem = $command.getProperty("Requires").getValue())
#foreach ($component in $components)
#if ($component.name == $sensor)
        return Robot::#variable($subsystem)->Get#class($component.name)()->PIDGet();
#end
#end
